downstream/modules/automation-hub/con-enable-view-only.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// assembly-view-only-access.adoc
+
+
+[id="con-enable-view-only"]
+
```

Comment:
Do you have to enable view-only access on initial installation of Private Hub, or can you update a running instance of hub? Just wondered, as the installation file is changed in this process.

Reply:
You can update existing installations by editing the inventory file and running the setup.sh script, which will update an existing AAP installation, according to the productization team.

Reply:
Do you think it would be worth mentioning this?  I assume it's somewhere in the docs already, but it might be useful.

Reply:
I think it is. Let me add a note explaining it